docs/workloads/azure/data/components/etl_pipelines/silver_data_azure.md

```diff
@@ -0,0 +1,45 @@
+# Silver processing
+
+Silver processing represents the 'Bronze to Silver' layer transformations. It accepts raw data
+(CSV files) as an input and converts it into a more structured delta format.
```

Comment:
i.e. CSV files - won't always be CSV files.

Reply:
done